src/diceBot/NeverCloud.rb

```diff
@@ -27,60 +27,44 @@ class NeverCloud < DiceBot
   # ダイスボットで使用するコマンドを配列で列挙する
   setPrefixes(['\d+NC.*', '\d+D6?([\+\-\d]*)>=\d+'])
 
-  def initialize
-    super
-  end
-
-  def changeText(string)
-    string
-  end
-
   def rollDiceCommand(command)
-    adjust = 0
-    adjust_str = ''
-    target = -1
-    target_str = ''
-    op = ''
-
-    case command
-    when /^\d+NC([\+\-\d]*)(>=)?(\d+)?$/i
-      adjust_str = Regexp.last_match(1) if Regexp.last_match(1)
-      adjust = parren_killer("(0#{adjust_str})").to_i if adjust_str != ''
-      op = Regexp.last_match(2) if Regexp.last_match(2)
-      target_str = Regexp.last_match(3) if Regexp.last_match(3)
-      target = target_str.to_i if target_str != ''
-      dice = command.to_i.to_s + 'D6' + adjust_str + op + target_str
-    when /^\d+D6?([\+\-\d]*)(>=)?(\d+)?$/i
-      adjust_str = Regexp.last_match(1) if Regexp.last_match(1)
-      adjust = parren_killer("(0#{adjust_str})").to_i if adjust_str != ''
-      target_str = Regexp.last_match(3) if Regexp.last_match(3)
-      target = target_str.to_i if target_str != ''
-      dice = command
-    else
-      return ''
-    end
-    total, dice_str, result_val, result = checkRoll(command.to_i, adjust, target)
-    if result != ""
-      return "(#{dice}) ＞ #{total}[#{dice_str}]#{adjust_str if adjust != 0} ＞ #{result_val} ＞ #{result}"
-    else
-      return "(#{dice}) ＞ #{total}[#{dice_str}]#{adjust_str if adjust != 0} ＞ #{result_val}"
+    m = /^(\d+)NC((?:[\+\-]\d+)*)(>=(\d+))?$/i.match(command) || /^(\d+)D6?((?:[\+\-]\d+)*)(>=(\d+))?$/i.match(command)
```

Comment:
2つの正規表現の共通部分が多いので、ひとつに合体させる方がむしろ読みやすく感じました。`||` でつないでいると、キャプチャの番号が2つの正規表現で共通であるかまで注意深く調べる必要があります。

また、文字クラスのエスケープが冗長で、短くできます。

```suggestion
    m = /^(\d+)(?:NC|D6?)((?:[-+]\d+)*)(>=(\d+))?$/i.match(command)
```